tests/cluecode/data/ics/chromium-net-tools-testserver/chromiumsync.py

```diff
@@ -6,4 +6,4 @@
     # to its nickname.
     self.clients = {}
     self.client_name_generator = ('+' * times + chr(c)
-        for times in xrange(0, sys.maxint) for c in xrange(ord('A'),ord('Z')))
+        for times in xrange(0, sys.maxsize) for c in xrange(ord('A'),ord('Z')))
```

Comment:
This is test data and not our own code. Please do not change this

Reply:
ok,I have done.